# HG changeset patch
# User Korotyha <kimalex@ngs.ru>
# Date 1416749630 -21600
#      Sun Nov 23 19:33:50 2014 +0600
# Node ID 913908fc9d4508b58586f19dec7acc78223e40a4
# Parent  01e5c993912c13719cdd0ba354f1ea4ba937d97c
Backout changeset 01e5c993912c13719cdd0ba354f1ea4ba937d97c

diff --git a/src/features.cpp b/src/features.cpp
--- a/src/features.cpp
+++ b/src/features.cpp
@@ -1219,7 +1219,7 @@
 #endif
 	  )
 	{
-		send_to_char("Невозможно использовать это так часто.\r\n", ch);
+		send_to_char("Не возможно использовать это так часто.\r\n", ch);
 		return;
 	}
 
